boards/arm/stm32f746g_disco/doc/stm32f746g_disco.rst

```diff
@@ -100,6 +100,8 @@ The Zephyr stm32f746g_disco board configuration supports the following hardware
 +-----------+------------+-------------------------------------+
 | GPIO      | on-chip    | gpio                                |
 +-----------+------------+-------------------------------------+
+| ETHERNET  | on-chip    | Ethernet                            |
```

Comment:
Can you update pinmuxing section too?

Reply:
Sure. Can you point be to a board doc and the relevant section, so i see what you are expecting?

Reply:
cf: https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nucleo_f429zi/doc/nucleof429zi.rst#default-zephyr-peripheral-mapping

Reply:
Should be fixed now